builder/builder-next/worker/worker.go

```diff
@@ -86,6 +86,10 @@ type Worker struct {
 	SourceManager *source.Manager
 }
 
+var _ interface {
+	GetRemotes(context.Context, cache.ImmutableRef, bool, cacheconfig.RefConfig, bool, session.Group) ([]*solver.Remote, error)
```

Comment:
While there's no exported interface to match yet, perhaps it would be good to add a "permalink" to the code in BuildKit for reference (to save people from hunting down the interface to look for)